File: backend/stock-serv/src/main/java/org/willingfish/stock/serv/entity/DealingRecordEntity.java
package org.willingfish.stock.serv.entity;

import lombok.Data;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;
import java.time.LocalDate;

@Entity
@Table(name = "DEALING_RECORD")
@Data
public class DealingRecordEntity extends BaseEntity {
    @Id @Column(name = "DEALING_RECORD_ID")
    Long id;
    String stockCode;
    String incomePrice;
    String outcomePrice;
    LocalDate incomeTime;
}

File: backend/stock-serv/src/main/kotlin/org/willingfish/stock/serv/entity/DealingRecordEntity.kt
package org.willingfish.stock.serv.entity

import java.time.LocalDate
import java.util.*
import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.Id
import javax.persistence.Table

@Entity
@Table(name = "DEALING_RECORD")
data class DealingRecordEntity(
        @Id @Column(name="DEALING_RECORD_ID")val id:Long?,
        val stockCode:String?,
        val incomePrice:String,
        val outcomePrice:String,
        val incomeTime:LocalDate,
        val outcomeTime: LocalDate,
        val createTime: Date,
        val updateTime: Date
)